cmdlet-tests-a2a.ps1

```diff
@@ -310,13 +310,27 @@ try {
    Write-Host "Create-AccessRequest"
    #region AccessRequest
    $selectedAccount = FindAssetAccount(0)
-   $selectedAsset = Get-SafeguardAsset -AssetToGet $selectedAccount.AssetId
+   $selectedAsset = Get-SafeguardAsset -AssetToGet $selectedAccount.Asset.Id
+   
+   Write-Host "Create-AccessRequest 2"

```

Comment:
Do you still need the debug output for WIP? (here & below)

Reply:
No, I don't. Thanks for noticing